src/System.Text.Json/System/Text/Json/JsonReaderException.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.Text.Json
+{
+    public class JsonReaderException : Exception
+    {
+        public JsonReaderException() : base() { }
```

Comment:
This was a placeholder for something that I haven't really figured out yet. Right now, we fail on a bad / unparsable stream. Long-term, I think it might be wiser to expose the errors in a different way.